egs/sad_rats/s5/RESULTS.txt

```diff
@@ -0,0 +1,121 @@
+command line (run on 2020 Nov 9 at 14:27:18) Version: 22  /home/john/kaldi_jjm/egs/sad_rats/s5/../../../tools/sctk/bin/md-eval.pl -1 -c 0.25 -r data/dev-1/rttm.annotation -s exp/sad_1a/dev-1/sad.rttm
```

Comment:
* Since the `sctk` path is already added in `path.sh`, perhaps you can replace the absolute path with just `md-eval.pl` so that it is easier for others to understand how to evaluate. 

* Since we only care about the missed speech and false alarm, you can just show those results by piping the output of md-eval to `awk '/(MISSED|FALARM) SPEECH/'`.

* Are you sure it is fair to evaluate with the `-1` and `-c 0.25` arguments. Recall that `-1` only evaluates single speaker regions (a SAD should detect speech even if more than 1 speakers are speaking), and 0.25 is a fairly permissive collar, especially considering that the annotations in RATS SAD corpus are done in 3 passes (including 2 manual passes). I leave this to your better judgement.